unit_tests/log_transfer/test_main_log_page_fetcher.cpp

```diff
@@ -174,3 +175,20 @@ const log_page *log_reader::get_page () const
 
 // Mock some of the functionality
 log_reader::log_reader () = default; // needed by log_page_fetch_task::execute
+
+PAGE_PTR
+pgbuf_fix_debug (THREAD_ENTRY *thread_p, const VPID *vpid, PAGE_FETCH_MODE fetch_mode, PGBUF_LATCH_MODE request_mode,
```

Comment:
Add an UT testcase for data page fetching. Optionally, as a separate PR. How do you prefer?

Reply:
I would add it as a separate task.